price-estimator/openapi.yml

```diff
@@ -137,10 +149,7 @@ components:
       description: If `baseunits` (the default) all amounts are denominated in the "natural" unit of the respective token given by the number of decimals specified through the ERC20 interface. If set to `atoms` all amounts are denominated in the smallest available unit (atom) of the token.
       required: false
       schema:
-        type: string
-        enum: [baseunits, atoms]
-        default: baseunits
-        example: baseunits
+        $ref: "#/components/schemas/Unit"
```

Comment:
This is because I noticed that the online editor has an extra table for schemas so that they can be viewed separately from the full routes.